cmd/kubeadm/app/phases/addons/dns/dns.go

```diff
@@ -139,11 +147,32 @@ func coreDNSAddon(cfg *kubeadmapi.MasterConfiguration, client clientset.Interfac
 	if err != nil {
 		return fmt.Errorf("error when parsing CoreDNS deployment template: %v", err)
 	}
+	kubeDNSConfigMap, err := client.CoreV1().ConfigMaps(metav1.NamespaceSystem).Get(kubeadmconstants.KubeDNS, metav1.GetOptions{})
```

Comment:
Could you please comment block above this line to outline why this here?

Reply:
Done.